src/components/Todolist.tsx

```diff
@@ -1,10 +1,86 @@
-import React from 'react';
-
-export const Todolist = () => {
-    return (
-        <div>
-
-        </div>
-    );
-};
-
+import React, { ChangeEvent, KeyboardEvent, useState } from 'react';
+import { FilterType, TasksType } from '../App';
+import './Todolist.css';
+
+export type TodolistType = {
+  title: string;
+  tasks: TasksType[];
+  removeTask: (id: string) => void;
+  filterTask: (filter: FilterType) => void;
+  addTask: (title: string) => void;
+  changeCheckStatus: (isDone: boolean, id: string) => void;
+  filter: FilterType;
+};
+export const Todolist = (props: TodolistType) => {
+  const [title, setTitle] = useState('');
+  const [eror, serEror] = useState<null | string>();
+
+  const addTaskHandler = () => {
+    if (title.trim() === '') {
+      serEror('Ошибка введите текст');
+    } else {
+      props.addTask(title);
+      setTitle('');
+    }
+  };
+  const onChangeHandler = (e: ChangeEvent<HTMLInputElement>) => {
+    setTitle(e.currentTarget.value);
+    serEror(null);
+  };
+
+  const onKeyPressHandler = (e: KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter') {
+      addTaskHandler();
+    }
+  };
+
+  return (
+    <div className={'wrapper'}>
```

Comment:
Если имя класса известно статически (т.е. берётся не из переменной, а хардкодится строкой) и оно единственное, можно писать чуть короче

```suggestion
    <div className="wrapper">
```

Reply:
@devlato, а можно использовать одинарные кавычки?